Dockerfile

```diff
@@ -14,7 +14,9 @@ RUN shellcheck -e SC1091,SC1090 ./*.sh
 FROM mcr.microsoft.com/dotnet/sdk:7.0.202-bullseye-slim AS restore
 WORKDIR /src
 COPY ./*.sln ./
-COPY */*.csproj ./
+# Using `Doppler.` prefix to avoid docker confuse symlink with directories
+# "ERROR: error from sender: readdir: readdirent Jenkinsfile: not a directory"
+COPY Doppler.*/*.csproj ./
```

Comment:
Hi @RodrigoPereyraDiaz!

After applying [this change](https://github.com/FromDoppler/doppler-jenkins/pull/30/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R36), this build started to fail.

It seems that current docker version is confusing the symlinks files `.doppler-ci` and `doppler-jenkins-ci.groovy` as directories and the line `COPY */*.csproj ./` was failing  with this error:

<img width="1163" alt="image" src="https://user-images.githubusercontent.com/1157864/229633031-99df01a2-e092-4ab7-9e5c-d2b11ea73a88.png">

This change fixes the issue, but we will need to apply the change in all repositories. Another option could be to copy explicitly each .csproj file to his folder.

I do not want to downgrade docker version, and I do not want to remove build neither because before that change I was getting this warning:

```
DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
Install the buildx component to build images with BuildKit:
https://docs.docker.com/go/buildx/
```

What do you think?

Reply:
Finally, I decided to remove buildx because it is breaking many builds.

https://github.com/FromDoppler/doppler-jenkins/pull/33